core/templates/dev/head/domain/question/EditableQuestionBackendApiService.ts

```diff
@@ -112,10 +113,46 @@ oppia.factory('EditableQuestionBackendApiService', [
       });
     };
 
+    var _addQuestionSkillLink = function(
```

Comment:
You don't need this function. Have addQuestionSkillLink call _addMultiQuestionSkillLinks while passing in the appropriate parameters. In general, reduce code duplication where possible.

Reply:
Done.

Reply:
Done.